donkeycar/parts/oak_d_camera.py

```diff
@@ -0,0 +1,359 @@
+import logging
+import time
+from collections import deque
+import numpy as np
+try:
+    import depthai as dai
```

Comment:
This should go into the RPi or Nano (wherever this is expected to work) section of setup.cfg.